### ISSUE TITLE
Export PCAP - wireshark cannot read the file

### ISSUE THREAD

**Issue by games130** [sipcapture/heplify-server#49]
I tried to export PCAP(from the WebUI) today which result in this error
Wonder if you have tried exporting pcap successfully?

I am using the latest heplify-server
![capture](https://user-images.githubusercontent.com/32865575/39418427-ea7a259e-4c8c-11e8-8b2c-0aec05135adc.JPG)

**Comment by negbie** [comment]
Hi,
yes I can export pcaps without any issues. Keep in mind heplify-server just fills the database and the export logic is inside the api.

**Comment by games130** [comment]
okay i seems i put my sentence in the wrong way. I can export the pcap successfully but when i try to open the pcap file i get the error.

**Comment by negbie** [comment]
Tested 6 different pcaps. IPv4+IPv6 no issues here.

**Comment by negbie** [comment]
Are you using the latest api?

**Comment by games130** [comment]
I am using the docker webhomer download few days ago from this project.

**Comment by games130** [comment]
@lmangani yup i understand this is not related heplify-server but i figured the API is matured so there should not be a problem there so i was thinking if it could be how the data is stored by heplify-server that could have cause this.

@negbie thank you for testing and verify that it is the docker build (not being the latest version)

**Comment by games130** [comment]
@negbie wondering if you have had the time to rebuild  the homer-webapp docker file?
I notice that people raised the same issue in homer-docker github. https://github.com/sipcapture/homer-docker/issues/63
